src/pages/LearningResources.js

```diff
@@ -1,231 +1,178 @@
+          tag='free'
+          description='A full Stack JavaScript developer from Canada I create free + premium courses and do a twice-weekly podcast called Syntax. HTML, CSS, JavaScript and WordPress web development tutorials! '
+          resourceurl='youtube.com/user/wesbos'
+        />
+
+        <ResourceCard
+          title='LevelUp Tutorials Youtube'
+          resourceimg='level-up.jpg'
+          tag='free'
+          description='Level Up Tutorials sets how to help fix a major problem in learning technologies. Videos created for Level Up Tutorials are aimed to be accessible and easy to follow while maintaining high production values.'
+          resourceurl='youtube.com/channel/UCyU5wkjgQYGRB0hIHMwm2Sg'
+        />
+
+        <ResourceCard
+          title='James Quick Youtube'
+          resourceimg='jamesquick.jpg'
+          tag='free'
+          description='Want to learn more about Web Development, Design, and Tools?  Then, youre in the right place! I LEARN the newest techniques in Web Development, Design, and Tools. I BUILD projects for myself and others.'
+          resourceurl='youtube.com/channel/UC-T8W79DN6PBnzomelvqJYw'
+        />
+
+        <ResourceCard
+          title='Level Up Financial Planning'
+          resourceimg='levelup-fin.jpg'
+          tag='free'
+          description='Fee only financial planner (fiduciary) virtually out of Fort Collins. Lucas can help you achieve confidence to take Your Financial Confidence to the Next Level.'
+          resourceurl='levelupfinancialplanning.com/videos/'
+        />
+
+        <ResourceCard
+          title='Codecademy (Freemium)'
+          resourceimg='codecademy.jpg'
+          tag='free'
+          description='Committed to building the best learning experience inside and out, making Codecademy the best place for our team to learn, teach, and create the online learning experience of the future.'
+          resourceurl='codecademy.com'
+        />
+
+        <ResourceCard
+          title='Traversy Media'
+          resourceimg='traversy.jpg'
+          tag='free'
+          description='Traversy Media features the best online web development and programming tutorials for all of the latest web technologies including Node.js, Angular 2, React.js, PHP, Rails, HTML, CSS and much more.'
+          resourceurl='youtube.com/user/TechGuyWeb'
+        />
+
+        <ResourceCard
+          title='Programming With Mosh'
+          resourceimg='mosh.jpg'
+          tag='free'
+          description='I train professional software engineers that companies love to hire. Javascript, React, React Native, Node, Angular, Python, C Sharp'
+          resourceurl='youtube.com/user/programmingwithmosh'
+        />
+
+        <div className='container justify-start my-4 px-8 border-b-1'>
+          <h3 className='text-xl text-blue-900 sm:text-2xl md:text-3xl'>Paid Resources</h3>
+        </div>
+
+        <div className='content-center flex grid-flow-col flex-wrap'>
```

Comment:
Same as the free resources change above. Add `justify-center` to the list of classes.

Reply:
Adjusted Paid Resources content div to be ```justify-center```

![paid](https://user-images.githubusercontent.com/14058890/88487973-010e2f80-cf58-11ea-8596-79669fc2250b.png)